manta-js/README.md

```diff
@@ -1,280 +1,27 @@
-# Manta JS
+# Manta SDK
 
-This package implements a Javascript SDK for connecting with the Manta Network.
+## Quick Start
 
-## Installation
+- ### If you want to connect Manta Wallet in your dApp, build private-related transactions.
+  - Docs: [how-to-use-manta-private-wallet-in-dapp](./docs/how-to-use-manta-private-wallet-in-dapp.md)
+  - Example: [extension-example](./examples/extension-example)
+  - Online Website: [https://2076b1.csb.app](https://2076b1.csb.app)
+  - Manta Wallet Download: [Chrome WebStore](https://chrome.google.com/webstore/detail/enabgbdfcbaehmbigakijjabdpdnimlg), [Beta Packages](https://github.com/Manta-Network/manta-extension/actions)
+- ### If you want to test the SDK or integrate the SDK into your wallet
+  - Docs: [how-to-use-manta-js-sdk](./docs/how-to-use-manta-js-sdk.md)
+  - Example: [sdk-example](./examples/sdk-example)
+- ### If you want to test zkSBT related functions
+  - Docs: [how-to-mint-zk-sbt](./docs/how-to-mint-zk-sbt.md)
 
-```sh
-yarn install manta.js
-```
-
-> If using sdk in a node.js environment please go to [Node Specific](#node-specific)
-
-### Local Development
-
-1. `git clone https://github.com/Manta-Network/sdk.git`
-2. `cd sdk/manta-js/package`
-3. `yarn`
-4. `yarn build-all` (Note: go to [Node Specific](#node-specific) if running in node.js environment)
-5. add `"manta.js": "file:/{LOCAL PATH OF sdk/manta-js/package}` to your project's package.json
-6. `yarn upgrade manta.js` in your project's directory
-
-# Usage
-
-All methods are called through the `MantaPrivateWallet` class.
-
-`manta-signer` must be installed and running.
-
-> If running `manta-signer` on dev mode, you should use the following features: `features=unsafe-disable-cors,disable-restart`.
-
-Refer to `/examples` for more thorough examples, and how to run them.
-
-## Node Specific
```

Comment:
Should keep some node documentation... We can open issue and do after this PR.

 Not sure if there is a way to connect to extension in node environment

Reply:
The node side should not require an extension, similar to the logic of sdk-example, add a node test example